src/runtime.js

```diff
@@ -323,20 +348,26 @@ let tornado = {
        return typeof val === 'function';
      },
 
-     /**
-      * Determine Tornado truthiness
-      * @param {*} val The value in question
-      * @return {Boolean}
-      */
-     isTruthy(val) {
-       if (val === 0) {
-         return true;
-       }
-       if (Array.isArray(val) && !val.length) {
-         return false;
-       }
-       return !!val;
-     }
+    /**
+     * Determine Tornado truthiness
+     * @param {*} val The value in question
+     * @return {Boolean}
+     */
+    isTruthy(val) {
+      if (val === 0) {
+        return true;
+      }
+      if (Array.isArray(val) && !val.length) {
+        return false;
+      }
+      return !!val;
+    }
+  },
+
+  attrFilters: {
+    'a[href]': function(val) {
+      return val.replace(/^javascript:.*/, '');
```

Comment:
This was meant to be an example of what is possible rather than a legitimate filter. I was hoping to get your input on what the real filters should look like.